www/gatsby-config.js

```diff
@@ -12,6 +12,8 @@
 module.exports = {
   siteMetadata: {
     title: 'React: A JavaScript library for building user interfaces',
+    description: 'A JavaScript library for building user interfaces',
```

Comment:
Should this be more specific / less redundant? (If it's only used by the RSS feed, should it maybe say something more about what the RSS feed is?)

Reply:
To make it less confusing I'm thinking we could add a "rssFeedDescription" property to siteMetadata (as opposed to the generic "description" property) and have the plugin use that instead:

```
         {
          site {
            siteMetadata {
              title
              rssFeedDescription
              description: rssFeedDescription 
              siteUrl
              site_url: siteUrl
            }
          }
        }
```

Reply:
Or, if we want to keep exactly like the previous RSS Feed, we could something like this:

```javascript
siteMetadata: {
    title: 'React: A JavaScript library for building user interfaces',
    rssFeedTitle: 'React'
    rssFeedDescription: 'A JavaScript library for building user interfaces',
    siteUrl: 'https://reactjs.org',
  },
```

```
{
          site {
            siteMetadata {
              rssFeedTitle
              title: rssFeedTitle
              rssFeedDescription
              description: rssFeedDescription 
              siteUrl
              site_url: siteUrl
            }
          }
        }
```

Reply:
This seems slightly prefable